yt_dlp/extractor/nerdcubed.py

```diff
@@ -1,33 +1,37 @@
-import datetime
-
 from .common import InfoExtractor
+from .youtube import YoutubeIE
+from ..utils import parse_iso8601, url_or_none
+from ..utils.traversal import traverse_obj
 
 
 class NerdCubedFeedIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?nerdcubed\.co\.uk/feed\.json'
+    _VALID_URL = r'https?://(?:www\.)?nerdcubed\.co\.uk/?(?:$|[#?])'
     _TEST = {
-        'url': 'http://www.nerdcubed.co.uk/feed.json',
+        'url': 'http://www.nerdcubed.co.uk/',
         'info_dict': {
             'id': 'nerdcubed-feed',
             'title': 'nerdcubed.co.uk feed',
         },
-        'playlist_mincount': 1300,
+        'playlist_mincount': 5500,
     }
 
-    def _real_extract(self, url):
-        feed = self._download_json(url, url, 'Downloading NerdCubed JSON feed')
+    def _entries(self, feed):
+        for feed_entry in traverse_obj(feed, ('videos', lambda _, v: v['id'])):
+            yield self.url_result(
```

Comment:
Personally, I'd remove one indentation like:
```suggestion
    def _extract_video(self, feed_entry):
        return self.url_result(
```
```py
        return self.playlist_result(
            map(self._extract_video, traverse_obj(feed, ('videos', lambda _, v: v['id']))),
            video_id, 'nerdcubed.co.uk feed')
```